### DISCUSSION TITLE
Increase size - zoom?

### DISCUSSION THREAD

**Discussion by twtonicr** [Fabrizz/MMM-OnSpotify#38] [General]
Hi, superb module!  I just upgraded to a Pi 4 and rebuilt my mirror, now using MMM-OnSpotify instead of MMM-NowPlayingOnSpotify

One question, I have my MM in portrait aspect, and its job is to show to thew entire room what is playing on spotify. How can I make the module display full width of the display?

**Comment by Fabrizz** [comment]
Hi, you could set the module position to `middle_center`, then in your `custom.css` file you should change the size of the module using CSS.
I would recommend changing the variable: `--ONSP-INTERNAL-PLAYER-SIZE-WIDTH`. Example:
```css
.ONSP-Custom { /* This class selects the module wrapper */
    --ONSP-INTERNAL-PLAYER-SIZE-WIDTH: 18em !important; /* or 1000px or 80% */
}

```

It also depends on the display aspect ration if it will show centered, you could change some CSS properties like making the global container `display: flex` for example, but as its just a fixed display I would just change the size and if its not centered I would just add some padding. Example:
```css
.ONSP-Custom {
    margin-bottom: 80px;  /* margin-top, margin-left, etc */
}
```

**Comment by twtonicr** [comment]
Thanks Fabrizz, sorry for the delay in replying.

I made the changes in:
~/MagicMirror/css/custom.css

(The css in ~/MagicMirror/modules/MMM-OnSpotify/css/custom.css seemed to have no effect)


`.ONSP-Custom { /* This class selects the module wrapper */
    --ONSP-INTERNAL-PLAYER-SIZE-WIDTH: 47em !important; /* or 1000px or 80% */
}`

--ONSP-INTERNAL-PLAYER-SIZE-WIDTH: 47em   worked OK for image, but text did not increase size.
--ONSP-INTERNAL-PLAYER-SIZE-WIDTH: 80%    the image disappeared completely, but the progress bar increased size. The font did not increase.

What I would like, is to scale the entire module, such that the image fills the complete width of the screen, and the fonts and progress bar all increase too.  There is nothing else on my magic mirror, its purpose is only spotify.

I tried to find a reference of options like "ONSP-INTERNAL-PLAYER-SIZE-WIDTH" to try and work it out, but  I couldn't find anything. Google didn't give any hits.  And help much appreciated.

**Comment by Fabrizz** [comment]
Hi I did not saw this. Let me try something, also, if you give me the resolution of your screen I can just simulate it
